Tiles/SandstoneSlabWall.cs

```diff
@@ -16,10 +15,9 @@ public override bool IsLoadingEnabled (Mod mod)
 		
 		public override void SetStaticDefaults()
 		{
-			Main.wallLargeFrames[Type] = (byte) 1;
+			Main.wallLargeFrames[Type] = 1;
 			Main.wallHouse[Type] = true;
 			DustType = DustID.Stone;
-			ItemDrop = ModContent.ItemType<Items.Placeable.SandstoneSlabWall>();
```

Comment:
How does tModLoader determine tile drops now?

Reply:
Here is the porting notes from Tmodloader

`ModTile.RegisterItemDrop to add fallback and manual type+style item drops`
by **jopojelly** has been merged.
**Pull Request:** <https://github.com/tModLoader/tModLoader/pull/3420>
**Arrives in Preview**: `Available now.`
**Arrives in Stable**: `With Stable Release of 1.4.4`
**Breakage**: 🟠 - **Medium**

**Short Summary**:
> - Reworks automatic tile drops introduced in <https://github.com/tModLoader/tModLoader/pull/3210>
> - `ModBlockType.ItemDrop` removed.
> - `TileLoader.GetItemDropFromTypeAndStyle` no longer falls back to tile style 0. A special fallback will be used if the style is not found.
> - `ModTile.RegisterItemDrop` added. Allows manually registering an item drop for a tile style, or as a fallback item. Used to provide drops for styles that don't have a corresponding item that places it.
> - `ModWall.RegisterItemDrop` added. Allows manual registration of wall drop, useful for unsafe walls that drop safe walls.

**Porting Notes**:
> - Run tModPorter. Remove `ItemDrop` lines. If any would be necessary, use `RegisterItemDrop` method to restore the functionality.
> - If mod has already been ported to 1.4.4 and is using `ModTile.GetItemDrops` already, consider using `ModTile.RegisterItemDrop` instead if `ModTile.GetItemDrops` was only used for providing drops for styles without corresponding items. See how `ExampleChest` was changed in the "Files Changed" tab.
> - If mod has already been ported to 1.4.4 and relied on the fallback behavior, use `RegisterItemDrop(itemtypehere)` to continue providing the previous fallback behavior.

seems as if it traverses to the item then back.